### PR TITLE
Add Code of Conduct page

### PR DESCRIPTION
**Pull request by jessbudd**
Added Code of Conduct page and styles for interior pages. Updated fenders logo footer link to return to homepage.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
     "react-emotion": "^9.2.12",
-    "react-helmet": "^5.2.1"
+    "react-helmet": "^5.2.1",
+    "yarn": "^1.21.0"
```
Comment (antonjb) [inline package.json]: Better to remove this as it isn't a dependency of the project

File: src/components/Footer/footer.js [hunk 2]
```diff
@@ -34,6 +42,11 @@ const Footer = () => (
                             Email us
                         </a>
                     </li>
+                    <li>
+                        <a href="/code-of-conduct" title="Code of conduct">
+                            Code of conduct
```
Comment (antonjb) [inline src/components/Footer/footer.js]: If the title is the same as the text, is it needed?

File: src/pages/code-of-conduct.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+                    ),
+                </li>
+                <li>
+                    Twitter Direct Message (
+                    <a href="https://twitter.com/fendersperth">
+                        https://twitter.com/fendersperth
+                    </a>
+                    )
+                </li>
+                <li>DM @mandy via the slack group</li>
```
Comment (antonjb) [inline src/pages/code-of-conduct.js]: Should we include all the organisers?

Comment (mandymichael) [reply src/pages/code-of-conduct.js]: I'd prefer we left it as just me for now please

File: src/components/Footer/footer.js (hunk 2 above)
Comment (jessbudd) [reply src/components/Footer/footer.js]: No it's not, but all the other links had it, so I followed the convention. Would prefer to remove the title text from all the footer links actually.

Comment (antonjb) [reply src/components/Footer/footer.js]: If you're happy to do that, then go for it. Will leave it up to you :)

**Comment by jessbudd** [comment]
Added Code of Conduct page and footer link

**Pull request merged**